_tests/e2e/tests/config/wdio.conf.base.js

```diff
@@ -8,7 +8,7 @@ const testType = typeof commandlineArgs.testType === 'undefined' ? 'export' : co
 const exclude = testType === 'export' ? 'drupal' : 'export';
 
 let host;
-if (commandlineArgs['base-url'].includes('developers-pr') && testType === 'drupal') {
+if (commandlineArgs['base-url'].includes('developers-pr.stage.redhat.com') && testType === 'drupal') {
```

Comment:
I thought we had a different URL

Reply:
@LightGuard Yep from now on URLs for preview environments will be `developer-preview-*.something.paas.redhat.com`. I'm just keeping this here so that should we need to back this out, it should keep working.